src/Components/Navbar.js

```diff
@@ -0,0 +1,47 @@
+import React  from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Tab from '@material-ui/core/Tab';
+import TabContext from '@material-ui/lab/TabContext';
+import TabList from '@material-ui/lab/TabList';
+import TabPanel from '@material-ui/lab/TabPanel';
+
+
+
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        flexGrow: 1,
+        backgroundColor: theme.palette.background.paper,
+    },
+}));
```

Comment:
Use `styled-components` instead of `makeStyles` solution of `material-ui`.
I looked it up, it's possible. look at these links:
[https://material-ui.com/guides/interoperability/](https://material-ui.com/guides/interoperability/)
[https://material-ui.com/guides/interoperability/#styled-components](https://material-ui.com/guides/interoperability/#styled-components)


You can learn more about `styled-components` online.